climpred/metrics.py

```diff
@@ -126,14 +134,84 @@ def _mae(forecast, reference, dim='svd', comparison=None):
     return mae(forecast, reference, dim=dim)
 
 
-def _crps(forecast, reference, dim='svd', comparison=None):
+def _brier_score(forecast, reference, **kwargs):
+    """Calculate Brier score for forecasts on binary reference.
+
+    ..math:
+        BS(f, o) = (f - o)^2
+
+    Args:
+        * forecast (xr.object)
+        * reference (xr.object)
+        * func (function): function to be applied to reference and forecasts
```

Comment:
If this is required, shouldn't it be an explicit argument then and not a keyword?